README.md

```diff
@@ -39,6 +39,7 @@ PGDATABASE=
 PGPASSWORD=
 PGPORT=
 SLACK_API_TOKEN=
+APPLICATIONS_OPEN_STATUS=
```

Comment:
I love that we're not just deleting the code for applications, but making the status a feature of our application.